oasislmf/model_preparation/lookup.py

```diff
@@ -310,6 +310,8 @@ def get_custom_lookup(
                 output_directory=output_directory
             )
 
+
+    ## Depricate Method - Load exposure from data.get_location_df()
     @classmethod
     def get_exposure(cls, lookup=None, source_exposure=None, source_exposure_fp=None):
```

Comment:
If depricated, should we remove it? Or is it still called by another function so we should remove it in the future? Or perhaps a third party uses this method?

Reply:
yup, removed